src/Sanitizer.php

```diff
@@ -16,8 +16,14 @@ class Sanitizer
      */
     private array $filters;
 
-    public function __construct()
+    private array $removeBlocks = ['script', 'iframe', 'object'];
+    
+    private array $removeAttributtes = [];
+
+    public function __construct(array $removeBlocks = [], array $removeAttributtes = [])
     {
+        $this->removeBlocks = [...$this->removeBlocks, ...$removeBlocks];
+        $this->removeAttributtes = [...$this->removeAttributtes, ...$removeAttributtes];
```

Comment:
This could be simplified as there's no default on the class. The class property doesn't need an initial value.